.gitignore

```diff
@@ -1,3 +1,7 @@
  .vagrant
 *.box
-.DS_Store
\ No newline at end of file
+.DS_Store
+node_modules
```

Comment:
I would rather define specific .gitignore for the folder where you are operating. 
Not sure if this "node_modules" defined here will be applicable for the subfolder

Reply:
Moved to the separated .gitignore inside new folder related to the Step Function